interview-app/src/test/java/ua/edu/ukma/interviewapp/InterviewAppApplicationTests.java

```diff
@@ -0,0 +1,14 @@
+package ua.edu.ukma.interviewapp;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class InterviewAppApplicationTests {
+
+    @Test
+    void contextLoads() {
+        System.out.println("Context loaded");
+    }
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def rcurly' has incorrect indentation level 4, expected level should be 2.
<!-- __reviewdog__:ChBiYTU2M2U1YjMxMTkwOGE5EgpjaGVja3N0eWxl -->